posts/2024-02-01-january-ecodev.mdx

```diff
@@ -0,0 +1,123 @@
+
+It's just the beginning for the Waku Network; we're still dogfooding it. You can see [the list of milestones and the roadmap for improvements](https://github.com/waku-org/pm/issues/101) put forward by the research team. There’s also an interesting discussion you can join in the [Vac Forum](https://forum.vac.dev/t/message-validation-marketplace-waku-network-of-networks/262/2).
+
+As part of Generation 0 of Waku Network, we have integrated sharding, ensuring that it’s scalable to thousands of users and that the bandwidth is reasonable for people running on Raspberry Pis or smaller devices with weaker connectivity. Our client targeted at low-resource devices, JSWaku, has been making progress towards fully supporting sharding and, by that, supporting the full Waku Network.
+
+
+### Future incentivisation for running Waku nodes?
+
+**Vaclav: **We are often asked about how we incentivise node runners, as well as who is running the nodes and why. There are currently no monetary or token incentives. However, the research team has recently published findings that offer a glimpse into an incentivisation proposal. That's very interesting and promising for the future of Waku node operators to maybe get something in return.
+
+
+### Growth of the Waku dev ecosystem
+
+**Guru: **We had our first Waku hackathon at ETHRome last year where 120 hackers participated. We received eight project submissions from EthRome, which was a high number for us as a first-time sponsor. At ETHIstanbul our submissions increased to 20. We closed the year with ETHIndia where we had a huge total of 51 project submissions.
+
+The graph has been trending consistently upwards as more hackers and developers discover Waku. We are exploring ways to offer support to these projects in terms of development, as well as raising funding and grants.
+
+
+### Project integrations with Waku
+
+**Guru: **Waku has been integrated into several existing projects already. [Status](https://status.app/), of course, is the original use case for Waku, and they have been the first to benefit from the Waku Network. 
+
+[Railgun](https://www.railgun.org/) has been friends of Waku for a long period of time, and they integrated Waku around a year and a half ago. We also completed an integration with [The Grap](https://thegraph.com/)h, which uses Waku in its Graphcast SDK to decentralise its indexes.
+
+[Silence Laboratories](https://www.silencelaboratories.com/) has also been researching building a multiparty computation (MPC) on top of Waku.
+
+[Tribes](https://www.tribes.xyz/) is also exploring how Waku can be implemented in their stack. Tribes are focused on privacy and censorship assistance, so it would be a great fit.
```

Comment:
Are they cool for us to mention them?

Reply:
Vaclav is checking with the Tribes team before we publish